README.md

```diff
@@ -1,2 +1,34 @@
-# booyah
-Online meow manager for students
+# back-end-skillcatch
+Online app for new employees
+
+Instalarea dependintelor necesare se face cu:
+
+```
+#!java
+
+> mvn install
+
+Reimport Maven in proiect.
```

Comment:
Aici e suficient sa zici doar mvn install . Apoi " just deploy the war in a tomcat or other container". Presupunem totusi ca cine vrea sa ruleze WS-ul stie despre cum se faca asta in mod normal :)